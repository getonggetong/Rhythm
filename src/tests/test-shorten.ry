/*	Shorten Test

	Tests the following:
	1. Quarter to eighth notes
	2. Whole to sixteenth
	3. Chord shortening
	4. Sequence shortening
*/

init() {
	
	print(A4 / 2);
	print(A4.1 / 16);
	print([A4,B4,C4] / 2);
	print([Ab4.8,B4.8,[C#4.8,D8]] / 2);

}