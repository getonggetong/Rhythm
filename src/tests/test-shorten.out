A4.8
A4.16
[A4.8,B4.8,C4.8]
[Ab4.16,B4.16,[C#4.16,D8.8]]
